frontend/packages/noobaa-storage-plugin/src/components/details-card/details-card.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/internal/components/dashboard/dashboard-card';
+import { DetailsBody, DetailItem } from '@console/internal/components/dashboard/details-card';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { InfrastructureModel } from '@console/internal/models';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+import { CephClusterModel } from '../../../../ceph-storage-plugin/src/models';
+
+const getInfrastructurePlatform = (infrastructure: K8sResourceKind): string =>
+  _.get(infrastructure, 'status.platform');
+const getCephVersion = (cephCluster: K8sResourceKind): string =>
+  _.get(cephCluster, 'spec.cephVersion.image');
+
+const NOOBAA_SYSTEM_NAME_QUERY = 'NooBaa_system_name';
+
+const cephClusterResource: FirehoseResource = {
+  kind: referenceForModel(CephClusterModel),
+  namespaced: true,
+  namespace: 'openshift-storage',
+  name: 'ceph-cluster',
+  isList: true,
```

Comment:
This doesn't seem like it should be a list?

Reply:
the ceph cluster resource is being returned in an array of size one

Reply:
@cloudbehl ^^

Reply:
@afreen23 I tried on your env and the CRD's instance name is called `rook-ceph`, not `ceph-cluster`. Is that expected ? What is the difference ? Storage dashboard is looking for `ceph-cluster` https://github.com/openshift/console/pull/1803/files#diff-d1f681bbd4c42188c00ffd77a9b11d73R40

Reply:
@rawagner The name of the storage cluster is not going to come from rook-operator and will not be rook-ceph. Right now for the reference, I have put ceph-cluster but I am not sure what will be the name once the OCS cluster is development ready. But yes if we are using rook operator we can change it to rook-ceph to the name.

The OCS operator is right now in the development phase and we don't have any CRD available for it. We need to change these few things once available.

Reply:
ok, but what should noobaa use ? `ceph-cluster` or `rook-ceph` ? 

The Firehose resource definition is incorrect. It should either be

```suggestion
 const cephClusterResource: FirehoseResource = {
  kind: referenceForModel(CephClusterModel),
  namespaced: true,
  namespace: 'openshift-storage',
  name: 'ceph-cluster',
  isList: false,
  prop: 'ceph',
```
or 

```suggestion
 const cephClusterResource: FirehoseResource = {
  kind: referenceForModel(CephClusterModel),
  namespaced: true,
  namespace: 'openshift-storage',
  name: 'rook-ceph',
  isList: false,
  prop: 'ceph',
```

`isList` should be false

Reply:
@rawagner I agree it should not use it. But We don't have a value to show inversion right now as it is going to be there from OCS operator CR's. The option is to put is as `-` right now or show some value which will then come from OCS operator. You can suggest us

@afreen23 my PR is already updated and tested with the changes @rawagner provided. https://github.com/openshift/console/pull/1803/files#diff-d1f681bbd4c42188c00ffd77a9b11d73R36

Reply:
@cloudbehl @rawagner  I have marked it as false as well and it is showing - .

Reply:
@afreen23 you need to change https://github.com/openshift/console/pull/1824/files#diff-e7956324aa3d22bd01ba7006f8a018feR93

`value={getCephVersion(_.get(cephClusterData, '0'))}` to `value={getCephVersion(cephClusterData)}`

Reply:
@rwagner changed